experimental/docker-stacks-and-bundles.md

```diff
@@ -109,7 +109,7 @@ are persisted as files, the file extension is `.dab` (Docker 1.12RC2 tools use
 A bundle has two top-level fields: `version` and `services`. The version used
 by Docker 1.12 tools is `0.1`.
 
-`services` in the bundle are the services that comprise the app. They
+`services` in the bundle is the services that comprise the app. They
```

Comment:
I think `services`  is a field name , like a proper noun. So it should be considered singular.